src/nvim/CMakeLists.txt

```diff
@@ -185,7 +193,11 @@ endfunction()
 # These lists must be mutually exclusive.
 foreach(sfile ${NVIM_SOURCES}
               "${CMAKE_CURRENT_LIST_DIR}/regexp_nfa.c"
-              ${GENERATED_API_DISPATCH})
+              ${GENERATED_API_DISPATCH}
```

Comment:
Interesting: I have not seen complains about us not using quotes in cmake files, but when I tried doing the same thing for https://github.com/latex-g7-32/latex-g7-32 I was short to receive complains about not compiling in directories with parenthesis (if I am not mistaking, this should be a problem for parenthesis only, maybe double quotes also; problems with semicolons are inevitable and directories with spaces are just fine with the current code).